projects/scripts/adi_project_xilinx.tcl

```diff
@@ -292,6 +292,8 @@ proc adi_project_files {project_name project_files} {
   foreach pfile $project_files {
     if {[string range $pfile [expr 1 + [string last . $pfile]] end] == "xdc"} {
       add_files -norecurse -fileset constrs_1 $pfile
+    } elseif [regexp "_constr.tcl" $pfile] {
```

Comment:
This lines should be in a separate commit. In order to track changes on adi_projects_xilinx.tcl

Reply:
Solved in this [commit](94f5c75db9c731db0053d3dd785d8770f03a370f).